src/shared/components/driverAnnotations/TierAnnotationControlsResultsView.tsx

```diff
@@ -0,0 +1,402 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import ErrorIcon from '../ErrorIcon';
+import autobind from 'autobind-decorator';
+import {
+    IDriverAnnotationControlsState,
+    IDriverAnnotationControlsHandlers,
+} from '../../alterationFiltering/AnnotationFilteringSettings';
+import {
+    EditableSpan,
+    DefaultTooltip,
+    getNCBIlink,
+} from 'cbioportal-frontend-commons';
+import 'rc-tooltip/assets/bootstrap_white.css';
+
+export interface IDriverAnnotationControlsProps {
+    state: IDriverAnnotationControlsState;
+    handlers: IDriverAnnotationControlsHandlers;
+    resultsView?: boolean;
+}
+
+enum EVENT_KEY {
+    distinguishDrivers = '0',
+    annotateOncoKb = '1',
+    annotateHotspots = '2',
+    annotateCBioPortal = '3',
+    annotateCOSMIC = '4',
+    customDriverBinaryAnnotation = '5',
+}
+
+@observer
+export default class TierAnnotationControlsResultsView extends React.Component<
```

Comment:
why is this called Tier and instead of more general?

Reply:
Renamed to DriverAnnotationControls.